lib/core/configurator.c

```diff
@@ -958,6 +958,15 @@ static int on_config_stash(h2o_configurator_command_t *cmd, h2o_configurator_con
     return 0;
 }
 
+static int on_config_usdt_selective_tracing(h2o_configurator_command_t *cmd, h2o_configurator_context_t *ctx, yoml_t *node)
+{
+    ssize_t on;
+    if ((on = h2o_configurator_get_one_of(cmd, node, "OFF,ON")) == -1)
+        return -1;
+    ctx->globalconf->usdt_selective_tracing = (uint8_t)on;
```

Comment:
```suggestion
    ctx->globalconf->usdt_selective_tracing = (int)on;
```

Assuming that the intent is to use `int` as being defined in the header file.